File: submissions/Arnab97/problem3.cpp

import java.io.*;
class problem3
{
    public static void main(String args [])throws IOException
    {
        InputStreamReader read = new InputStreamReader(System.in);
        BufferedReader ob = new BufferedReader(read);
        System.out.println("Enter the number of test cases");
        int c = Integer.parseInt(ob.readLine());
        double print[] = new double[c];
        int k = 0;
        String w = "";
        String w1 = "";
        int a = 0,b = 0;
        int arr[] = new int[2];
        int l = 0,z = 0;
        for (int i=0;i<c;i++)
        {
            String s = ob.readLine ();
            for (int j=0;j<s.length();j++)
            {
                char ch = s.charAt(j);
                w = "";
                if (ch!=' ')
                    w = w + ch;
                else
                {
                    arr[l++] = Integer.parseInt(w);
                    System.out.println(w);
                    j++;
                }
            }
            double e = Math.pow(arr[0],arr[1]);
            print[k++] = e%10;
        }
        for (int i=0;i<2;i++)
        {
        System.out.println(print[i]);
        }
    }
}

File: submissions/ankitbiswas/problem3.java
import java.io.*;
class problem3
{BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
 int t=0;int a=0;int b=0;
 void main()throws IOException
 {System.out.println("Enter test cases");
   t=Integer.parseInt(br.readLine());int ar[]=new int[t];int arr[]=new int[t];
  for(int i=1;i<=t;i++)
    {a=Integer.parseInt(br.readLine());
     b=Integer.parseInt(br.readLine());
     int s=(int)(Math.pow(a,b));int c=s;int coun=0;
     
     arr[i-1]=s%10; 
    }
  for(int i=1;i<=t;i++)
   {System.out.println(arr[i-1]);}
    
  }}
    
  